### PR TITLE
Improve Japanese readability

### PR DESCRIPTION
**Pull request by emergent-reasons**
Translator did a lot of work on this one. It should have a good impact on readability of Japanese by gluing words together so that browsers don't insert line breaks in the middle of words. There are also some grammar and vocabulary changes for the same purpose. It will be more time consuming to change the translation in the future but that is not going to happen often.

It had a side effect of converting a number of character codes like `&quot;` back to regular characters, due to change of braces from `{{...}}` to `{{{...}}}`.

### CODE REVIEW

**Comment by emergent-reasons** [comment]
@zquestz This one is ready for review. It's basically a lot of additions of the special <span> class.

File: .build/faqs/index.html [hunk 1]
```diff
@@ -108,7 +108,7 @@ <h2 class="faqsBlock-heading">Overview</h2>
         <p><span class="leading">Q</span>What is the difference between coin mixing and CashShuffle?</p>
       </div>
       <div class="faqs-a">
-        <p><span class="leading">A</span>&quot;Coin mixing&quot; commonly refers to the use of services that allow a user to replace his or her coins with a different set of coins. CashShuffle is different. It allows users to combine their transactions with others, creating obfuscation.</p>
+        <p><span class="leading">A</span>"Coin mixing" commonly refers to the use of services that allow a user to replace his or her coins with a different set of coins. CashShuffle is different. It allows users to combine their transactions with others, creating obfuscation.</p>
```
Comment (zquestz) [inline .build/faqs/index.html]: Hmm this is invalid HTML... these should be `&quot;` =\

Comment (zquestz) [reply .build/faqs/index.html]: Since you are using `{{{` we need to make sure strings are HTML escaped correctly.

Comment (emergent-reasons) [reply .build/faqs/index.html]: Ok I fixed everything within my knowledge including the English and Japanese translations and the explanation in the translation documentation.

The switch to `{{{` raw strings in a lot of places may make other translations more difficult. I will keep an eye on it.

**Comment by zquestz** [comment]
This looks great! Thanks @emergent-reasons

**Pull request merged**